kubernetes-utils.el

```diff
@@ -42,6 +42,20 @@ Update the pod state if it not set yet."
                                "\\1:\\2"
                                tz))))
 
+(defun kubernetes-get-pod-container-names (pod)
+  "Return the names of all containers available in the specified pod."
+  (let ((containers (cdr (assoc 'containers (assoc 'spec pod)))))
```

Comment:
It would be more in keeping with the style of this codebase to use the destructuring let and lambda provided by dash: 

```lisp
(-let [(&alist 'spec (&alist 'containers containers)) pod]
  (-map (-lambda ((&alist 'name name)) name) containers))
```

But it's not a big deal if you don't want to make the change.

Reply:
Sure, I'll make the change some time this evening!

Personally I think that the `dash`-version of this destructuring binding is less readable than the Clojure-version it is inspired by, but it makes sense to keep the code base in one style.

Reply:
I've amended the last commit